pkgs/top-level/perl-packages.nix

```diff
@@ -10616,12 +11202,10 @@ let
     checkInputs = [ pkgs.cairo ];
     propagatedBuildInputs = [ pkgs.gobject-introspection Glib ];
     meta = {
-      broken = true; # TODO: tests failing because "failed to load libregress.so"
```

Comment:
to preserve commens, they should be outside (above or below) whole meta section.
I am not sure they are  up to date, so let them go